### DISCUSSION TITLE
Any plans on supporting masked conditioning?

### DISCUSSION THREAD

**Discussion by JasonS09** [ssitu/ComfyUI_UltimateSDUpscale#31] [Q&A]
Title. There are several times already I have been in the need of masking conditioning for composition purposes (i.e to avoid bleeding to an extent), however it seems these are not supported yet. I think it would be a nice addition.

**Comment by ssitu** [comment]
You want to do this during the upscaling? Wouldn't you want to do the masking when you make the first image?

**Reply by JasonS09** [reply]
Because when I upscale with prompt and let's say, I need one character with red eyes and one with green eyes. Putting the conditioning in the whole image will result in mixed eyes from both characters even at 0.30 denoise. Defeating the purpose of having made the mask for the first image.

**Reply by ssitu** [reply]
Maybe a lower denoise can preserve the colors? The mask conditioning support is there, but I’ve never been able to test it properly because I never had a good use case for it.
